tests/unit_tests/synthetic/libraries/cutensor2hiptensor.cu

```diff
@@ -139,6 +141,16 @@ int main() {
   cutensorWorksizePreference_t tensorWorksizePreference_t;
   cutensorWorksizePreference_t TENSOR_WORKSPACE_MIN = CUTENSOR_WORKSPACE_MIN;
   cutensorWorksizePreference_t TENSOR_WORKSPACE_MAX = CUTENSOR_WORKSPACE_MAX;
+
+  // CUDA: const char* cutensorGetErrorString(const cutensorStatus_t error);
+  // HIP: const char* hiptensorGetErrorString(const hiptensorStatus_t error);
+  // CHECK: err = hiptensorGetErrorString(status);
+  err = cutensorGetErrorString(status);
+
+  // CUDA: size_t cutensorGetCudartVersion();
+  // HIP: int hiptensorGetHiprtVersion();
+  // CHECK: ver = hiptensorGetHiprtVersion();
+  ver = cutensorGetCudartVersion();
```

Comment:
Potential data lost in the return value. We'll need to warn about it. Please take note for future fixing.